packages/dappmanager/src/modules/installer/runPackages.ts

```diff
@@ -54,13 +55,20 @@ export async function runPackages(
 
       // To clean-up changing multi-service packages, remove orphans
       // but NOT for core packages, which always have orphans
-      const removeOrphans = !pkg.isCore;
 
       log(pkg.dnpName, "Starting package... ");
-      await dockerComposeUp(pkg.composePath, {
-        removeOrphans,
-        timeout: pkg.dockerTimeout
-      });
+      const removeOrphans = !pkg.isCore;
+
+      pkg.runningContainers
```

Comment:
Here https://github.com/dappnode/DNP_DAPPMANAGER/pull/472/files#diff-3e90a91864c6d48793b01d80c162240f3c31f12737695cba3b5039104553f96eR72
`runningContainers` will always be defined so this condition won't work

Reply:
![Screenshot from 2020-12-07 10-36-58](https://user-images.githubusercontent.com/41727368/101334452-2bfa7f80-3878-11eb-949e-c931125bd2ac.png)

runningContainers is possibly undefined

Reply:
According to typescript yes, but it will be undefined if the package is not found. If the package is found and all containers are running, `runningContainers` will be defined and contain all service names